falcon_kit/mains/ovlp_to_graph.py

```diff
@@ -777,17 +819,21 @@ def generate_string_graph(args):
 
     sg.init_reduce_dict()
 
-    #if not args.disable_chimer_prediction:
-    #    sg.mark_chimer_edges()
-    #sg.mark_spur_edge()
-
-
     sg.mark_tr_edges() # mark those edges that transitive redundant
 
     if DEBUG_LOG_LEVEL > 1:
         print sum( [1 for c in sg.e_reduce.values() if c == True] )
         print sum( [1 for c in sg.e_reduce.values() if c == False] )
 
+    if not args.disable_chimer_bridge_removal:
+        chimer_nodes, chimer_edges = sg.mark_chimer_edges()
+
+        with open("chimers_nodes", "w") as f:
+            for n in chimer_nodes:
+                print >>f, n
+    else:
```

Comment:
del chimer_nodes
    else:
      chimer_edges = set()

By unbinding `chimer_nodes` we make it clear to the reader that it's not needed elsewhere. And it's good for symmetry with the `else` block.